Signal/src/Models/MessageActions.swift

```diff
@@ -67,66 +67,67 @@ struct MessageActionBuilder {
     }
 }
 
-extension ConversationViewItem {
+@objc
```

Comment:
This was an expedient, but ultimately not very good use of `extension` that I regret. 

It was pretty straight forward, but was the only non-mechanical change related to moving ConversationViewItem to be a protocol.

Reply:
I'm curious why you regret it?

Reply:
I think extensions work best when the methods are dead simple and used all over the place.

But they aren't super flexible - e.g. you can't add methods to a protocol exported to objc, because objc doesn't have "default protocol implementations" like swift does.